1.6/Dockerfile.windows

```diff
@@ -0,0 +1,12 @@
+FROM windowsservercore
```

Comment:
@jhowardmsft that makes me kind of sad, and I hope there are plans to change that because being able to `docker run -it --rm ubuntu` on Windows sounds really magical and I want it to be a thing :innocent: